src/rdb/rdb.c

```diff
@@ -866,21 +866,28 @@ rdb_chkpt(struct rdb *db)
 static void
 rdb_chkptd(void *arg)
 {
+	struct timespec          last;
+	struct timespec          deadline;
 	struct rdb *db = arg;
 	struct rdb_chkpt_record *dcr = &db->d_chkpt_record;
 
 	D_DEBUG(DB_MD, DF_DB ": checkpointd starting\n", DP_DB(db));
+	clock_gettime(CLOCK_REALTIME, &last);
```

Comment:
Looks like here we could use MONOTONIC or even the coarse version of MONOTONIC.

Reply:
I used the example from the ABT website.   What happens if the REALTIME and MONOTONIC clocks get out of sync?

Reply:
Argobots internally uses REALTIME so I think we have to use it, no?   If I read the manpage correctly, it looks like MONOTONIC isn't affected by changes to the system time but REALTIME is so they may not be in sync.   We should probably request ABT change that.

Reply:
But I guess if REALTIME is a system call, this is pretty horrid.   Maybe we can check at startup if they are in sync and fail if they are not?

Reply:
I settled on CLOCK_REALTIME_COARSE after checking differences on my local system.